docs/content/_snippets/about-iota/ledger_devices.mdx

```diff
@@ -0,0 +1,9 @@
+::: info Supported Ledger Devices
+
+| Device             | Minimum Firmware Version | Notes |
+|--------------------|--------------------------|-------|
+| Ledger Nano S      | 0.9.2                    | The app on the this device has to be [installed manually](#manually-for-deprecated-devices-such-as-ledger-nano-s) because it is deprecated. |
+| Ledger Nano S Plus | 0.9.3                    | |
+| Ledger Nano X      | 0.9.3                    | |
+
```

Comment:
Maybe also list stax/flex as supported soon?

Reply:
Once they are, yet 👍

Reply:
Sure I can do that

Reply:
Done